backend/tournesol/views/rate_later.py

```diff
@@ -64,6 +65,36 @@ def perform_create(self, serializer):
         self.prefetch_entity(rate_later)
 
 
+@extend_schema_view(
+    post=extend_schema(
+        description="Add a multiple new entities to the logged user's rate-later list,"
+        " for a given poll.",
+        request=RateLaterSerializer(many=True),
+        responses={
+            201: RateLaterSerializer,
+        },
+    ),
+)
+class RateLaterBulkCreate(RateLaterQuerysetMixin, generics.CreateAPIView):
+    """
+    Create multiple rate-later entries at once.
+    Accepts an array of entities to be added to the rate-later list.
+    """
+
+    permission_classes = [IsAuthenticated]
```

Comment:
This following suggestion will disable the `limit` and `offset` query parameters automatically added by DRF.

```suggestion
    pagination_class = None
    permission_classes = [IsAuthenticated]
```

Reply:
Done in d4f78c4ce7fabd94e0ba97c651ac65cc9b40b6a6.